tests/lib/rules/no-inline-comments.js

```diff
@@ -88,7 +88,24 @@ ruleTester.run("no-inline-comments", rule, {
                comment
             */}
             </div>
-        )`
+        )`,
+
+        // Webpack
+        {
+            code: `
+              import(
+                /* webpackInclude: /\\.json$/ */
+                /* webpackChunkName: "my-chunk-name" */
+                './locale/en'
+              );
+            `,
+            options: [
+                {
+                    ignorePattern: "(?:webpackChunkName|webpackInclude):\\s.+"
+                }
+            ],
+            parserOptions: { ecmaVersion: 2020 }
```

Comment:
Can we also add some `invalid` examples where the comment doesn't match the configured pattern